compose/versions

```diff
@@ -1,5 +1,5 @@
 export OPENBALENA_API_VERSION_TAG=v0.19.5
 export OPENBALENA_DB_VERSION_TAG=v2.0.3
-export OPENBALENA_REGISTRY_VERSION_TAG=v2.7.0
+export OPENBALENA_REGISTRY_VERSION_TAG=v2.11.0
```

Comment:
This needs to move to whatever the final merged version for https://github.com/balena-io/open-balena-registry/pull/83 will be :) ).

Reply:
yes :)

Reply:
@hedss done - new version is `v2.11.1`